System/Constants.h

```diff
@@ -57,7 +57,49 @@ namespace RTE {
 	};
 
 	enum DotGlowColor { NoDot, YellowDot, RedDot, BlueDot };
-	enum TransparencyPreset { LessTrans, HalfTrans, MoreTrans };
+
+	enum DrawBlendMode {
+		NoBlend,
+		BlendBurn,
+		BlendColor,
+		BlendDifference,
+		BlendDissolve,
+		BlendDodge,
+		BlendInvert,
+		BlendLuminance,
+		BlendMultiply,
+		BlendSaturation,
+		BlendScreen,
+		BlendTransparency,
+		BlendModeCount
+	};
+
+	enum BlendAmountLimits { MinBlend = 0, MaxBlend = 100 };
+
+	enum TransparencyPreset {
```

Comment:
Why bother having this transparency preset enum? Is it any better than just passing the raw value?